main.py

```diff
@@ -13,6 +13,11 @@
 from requests import Response
 from requests import get as requests_get
 from sv_ttk import set_theme
+from webbrowser import open as openwebpage
+
+from dialogs import ask_yes_no
```

Comment:
We only use a single one? Remove the rest then, no?

Reply:
why? those are pretty small predefined functions that have barely any impact on the code size because they only make one call to the class. and maybe we will use other ones in the future